pkg/document/document.go

```diff
@@ -219,7 +219,12 @@ func (d *Document) ApplyChangePack(pack *change.Pack) error {
 		d.GarbageCollect(pack.MinSyncedVersionVector)
 	}
 
-	// 05. Update the status.
+	// 05. Remove detached client's lamport from version vector if it exists
+	if pack.MinSyncedVersionVector != nil {
+		d.doc.changeID = d.doc.changeID.SetVersionVector(d.doc.changeID.VersionVector().Filter(pack.MinSyncedVersionVector.Keys()))
+	}
```

Comment:
The same logic also exists in `internal_document`. It seems necessary to check why it is implemented repeatedly.

Reply:
oh okay, I'll figure it out and remove this if it's unnecessary

Reply:
Let’s see if it can be resolved after merging.